tools/cldr-apps/js/src/esm/cldrProgress.mjs

```diff
@@ -361,18 +367,10 @@ function fetchLocaleData(unlessLoaded) {
   if (!locale || locale === "USER") {
     return; // no locale
   }
-  if (
-    unlessLoaded &&
-    localeProgressStats &&
-    localeProgressStats.locale === locale
-  ) {
+  if (unlessLoaded && localeProgressStats?.locale === locale) {
     // LocaleMeter is already set
-    // Still refresh if it's been long enough since last request and last response
-    const now = Date.now();
-    if (
-      now < timeLastReceivedLocaleData + LOCALE_METER_REFRESH_MS &&
-      now < timeLastRequestedLocaleData + LOCALE_METER_REFRESH_MS
```

Comment:
It was my mistake to use && in this condition; should be || as it is in the new classe